account-lookup-service/chart-admin/values.yaml

```diff
@@ -5,7 +5,7 @@
 image:
   registry: docker.io
   repository: mojaloop/account-lookup-service
-  tag: v15.2.3
+  tag: v15.4.0-snapshot.29
```

Comment:
Let's release accoun-lookup-service first and include here the proper release tag.